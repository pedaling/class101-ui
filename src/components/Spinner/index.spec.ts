import { Spinner } from './';

describe('Grid', () => {
  it('값이 존재한다.', () => {
    expect(Spinner).toBeTruthy();
  });

  it('스냅샷과 일치한다..', () => {
    expect(Spinner).toMatchSnapshot();
  });
});
