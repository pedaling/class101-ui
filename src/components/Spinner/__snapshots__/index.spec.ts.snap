// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid 스냅샷과 일치한다.. 1`] = `
Object {
  "$$typeof": Symbol(react.memo),
  "compare": null,
  "defaultProps": Object {
    "backgroundColor": "#3e4042",
    "color": "rgba(0, 0, 0, 0.07)",
    "size": 24,
  },
  "type": [Function],
}
`;
